tests/test_serverless/test_modules/test_ping.py

```diff
@@ -57,7 +57,7 @@ async def test_start_ping(self, mock_get_return):
         mock_session.headers.update = Mock()
 
         # Success case
-        await new_ping._send_ping() # pylint: disable=protected-access
+        await new_ping.start_ping(test=True)
```

Comment:
Great improvement. Tests should definitely use the exposed methods, and not methods meant to be private/protected.